src/store/modules/movies.module.ts

```diff
@@ -86,6 +86,7 @@ const moviesModule: StoreonModule<IState, IEvents> = (store) => {
     let movies: Array<IMovie> = [];
     const queryParams = Object
       .entries(params)
+      // eslint-disable-next-line @typescript-eslint/no-unused-vars
       .filter(([key, value]) => Boolean(value))
```

Comment:
```suggestion
      .filter(([, value]) => Boolean(value))
```
?

Reply:
Possible option but looks strange to me :)
I would prefer to stay with explicit unused variable instead of void syntax.